src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 100500; i++) {
```

Comment:
100500 it was just an example in mentors comment) 
Let's create a constant 
private static final int NUMBER_OF_BALLS = 3; and use it here